client/src/components/App.js

```diff
@@ -1,57 +1,63 @@
-import Login from "./pages/auth/Login";
-import PrivateRoute from "./private-route/PrivateRoute";
-import Dashboard from "./pages/dashboard/Dashboard";
+import Navbar from './layout/Navbar';
+import Footer from './layout/Footer';
+import Landing from './pages/Landing';
+import Register from './pages/auth/Register';
+import Login from './pages/auth/Login';
+import PrivateRoute from './private-route/PrivateRoute';
+import Dashboard from './pages/dashboard/Dashboard';
 
 import '../assets/stylesheets/App.scss';
 
 // Check for token to keep admin logged in
 if (localStorage.jwtToken) {
-  // Set auth token header auth
-  const token = localStorage.jwtToken;
-  setAuthToken(token);
-  // Decode token and get admin info and exp
-  const decoded = jwt_decode(token);
-  // Set admin and isAuthenticated
-  store.dispatch(setCurrentAdmin(decoded));
-  // Check for expired token
-  const currentTime = Date.now() / 1000; // to get in milliseconds
-  if (decoded.exp < currentTime) {
-    // Logout admin
-    store.dispatch(logoutAdmin());
+    // Set auth token header auth
+    const token = localStorage.jwtToken;
+    setAuthToken(token);
+    // Decode token and get admin info and exp
+    const decoded = jwt_decode(token);
+    // Set admin and isAuthenticated
+    store.dispatch(setCurrentAdmin(decoded));
+    // Check for expired token
+    const currentTime = Date.now() / 1000; // to get in milliseconds
+    if (decoded.exp < currentTime) {
+        // Logout admin
+        store.dispatch(logoutAdmin());
 
-    // Redirect to login
-    window.location.href = "./login";
-  }
+        // Redirect to login
+        window.location.href = './login';
+    }
 }
 class App extends Component {
-  render() {
-    return (
-      <Provider store={store}>
-        <Router>
-          <div className="App">
-            <Navbar />
-            <Route exact path="/" component={Landing} />
-            <Route exact path="/register" component={Register} />
-            <Route exact path="/login" component={Login} />
-            <Switch>
-              <PrivateRoute exact path="/dashboard" component={Dashboard} />
-            </Switch>
-          </div>
-        </Router>
-      </Provider>
-    );
-  }
+    render() {
+        return (
+            <Provider store={store}>
+                <Router>
+                    <div className="App wrapper">
+                        <Navbar />
+                        <Route exact path="/" component={Landing} />
+                        <Route exact path="/register" component={Register} />
+                        <Route exact path="/login" component={Login} />
+                        <Switch>
+                            <PrivateRoute
+                                exact
+                                path="/dashboard"
+                                component={Dashboard}
+                            />
+                        </Switch>
+                        <Footer />
```

Comment:
This is probably the wrong place to comment, but you have to scroll down to the see the footer, also the footer button colors seem a bit transparent

Reply:
I know, I like it this way because it isn't on the screen, cluttering it up, but it still exists.And the buttons are just light gay because that's what looks good.